src/CoreFx.Private.TestUtilities/src/CoreFx.Private.TestUtilities.csproj

```diff
@@ -8,6 +8,7 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <NoWarn>$(NoWarn);CS1573</NoWarn>
     <EnablePInvokeAnalyzer>false</EnablePInvokeAnalyzer>
+    <RunMatchingRefApiCompat>false</RunMatchingRefApiCompat>
```

Comment:
NIT: Should we add a comment here saying why we want to exclude the reverse apicompat run? I only ask because it seems like if we did we would have catched the API you are adding right now on the TestUtitlities.cs ref code.

Reply:
It doesn't consistently expose API across all its configurations.  I tried to fix it in one case but found I couldn't fix it everywhere without adding baselines or removing API from some frameworks.  Since we don't care so much about this thing being used externally I felt that API not being present for tests to consume would be a good enough guard against folks forgetting to expose it.